Classes/Projection/ProjectorInterface.php

```diff
@@ -34,4 +34,11 @@ public function getReadModelClassName(): string;
      * @api
      */
     public function reset();
+
+    /**
+     * Returns true if the projection maintained by the concreted projector does not contain any data (yet).
+     *
+     * @return boolean
+     */
+    public function isEmpty();
```

Comment:
I'd also be fine with `count()` but @albe had concerns that it's not feasible to implement for all kinds of projectors. Another option might be `containsData`– but that's also a bit awkward.

I think `isEmpty()` is fine.